manifests/agent/install.pp

```diff
@@ -1,8 +1,7 @@
 # Install the puppet client installation
 class puppet::agent::install {
-
-  package { $puppet::params::client_package:
-    ensure => $::puppet::version,
+  package { $puppet::client_package:
```

Comment:
Based on params not being passed directly to this class or any other that I can find, it seems the standard throughout this module is to use `$puppet::some_param_name`. I thought the `::params::` that was in here before was my doing... guess I actually found a bug while modifying this manifest.